java/ql/lib/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -284,7 +288,8 @@ module ModelValidation {
     )
     or
     exists(string kind | sourceModel(_, _, _, _, _, _, _, kind, _) |
-      not kind = ["remote", "contentprovider", "android-widget", "android-external-storage-dir"] and
+      not kind =
+        ["remote", "sql", "contentprovider", "android-widget", "android-external-storage-dir"] and
       not kind.matches("qltest%") and
```

Comment:
So, this means that sources that start with `qltest` are always valid?

Reply:
Yes, that is correct. This is not ideal for production and basically just included because we validate models used in the external models test.